medcat/cat.py

```diff
@@ -201,7 +201,8 @@ def create_model_pack(self, save_dir_path: str, model_pack_name: str = DEFAULT_M
         _save_dir_path = save_dir_path
         save_dir_path = os.path.join(save_dir_path, model_pack_name)
 
-        os.makedirs(save_dir_path, exist_ok=True)
+        # expand user path to make this work with '~'
+        os.makedirs(os.path.expanduser(save_dir_path), exist_ok=True)
```

Comment:
Hi @Imipenem, was thinking you will add pathlib, is there a reason you changed?

Reply:
Hey, yeah see my pull request notes. 
I thought adding pathlib would be beneficial, but in this particular case, it would be just replacing `os.path` with the `pathlib` equivalent. I decided to stick with `os.path` since its used everywhere else in MedCat and I didn't wanted to break consistency just for this particular feature. 

I could, of course, add the `pathlib` equivalent instead but due to the reasons above, I thought sticking with `os.path` would be better.

Reply:
sorry about that, missed it, all good then, will merge